ChangeLog.d/9618.txt

```diff
@@ -0,0 +1,16 @@
+Removals
+   * MBEDTLS_NO_PLATFORM_ENTROPY and previously deprecated
+     MBEDTLS_ENTROPY_HARDWARE_ALT are removed. See documentation of
+     MBEDTLS_PLATFORM_GET_ENTROPY_ALT for a description on how entropy module
+     gathers entropy data.
```

Comment:
Minor: “and **the** previously deprecated”, “See **the** documentation”, “how **the** entropy module”...